tests/conftest.py

```diff
@@ -60,12 +65,33 @@ def python_conda_mirror_main_conda_forge_environment(tmpdir_factory):
     fn.write(content)
     return fn
 
+
+@pytest.fixture(scope="function")
+def python_conda_failing_env(tmpdir_factory):
+    content = """name: failing_env
+channels:
+- main
+- conda-forge
+dependencies:
+- python=3.9.5
+- sadfljdsajlfdalkjfdsjlfdsjdskf=10
+"""
+    fn = tmpdir_factory.mktemp("minimal_env").join("env.yml")
+    fn.write(content)
+    return fn
+
+
 # conda-locks LockSpecification object
 @pytest.fixture(scope="function")
-def lock_spec_fixture(python_conda_mirror_main_conda_forge_environment) -> LockSpecification:
-    lock_spec = get_lock_spec_for_environment_file(python_conda_mirror_main_conda_forge_environment)
+def lock_spec_fixture(
+    python_conda_mirror_main_conda_forge_environment,
+) -> LockSpecification:
+    lock_spec = parse_environment_file(
```

Comment:
see above

Reply:
was never used and needs to be deleted.  in progress.  (For now it's commented out).